README.md

```diff
@@ -25,6 +31,7 @@ This is the official [ownCloud](https://owncloud.com) server image. This Docker
 
 - [`latest`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:latest`
 - [`10.7.0`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:10.7.0`, `owncloud/server:10.7`, `owncloud/server:10`
+- [`10.6.0`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:10.6.0`, `owncloud/server:10.6`, `owncloud/server:10`
```

Comment:
`owncloud/server:10` is an "alias" for `owncloud/server:10.7` and not for `owncloud/server:10.6` anymore

Reply:
Obviously :+1: thanks!